07.bitstream.semantics.md

```diff
@@ -1314,6 +1315,41 @@ coded video sequence.
 number of bits used to represent time_offset_value is equal to time_offset_length. When
 time_offset_value is not present, its value is inferred to be equal to 0.
 
+#### Metadata crop semantics
+
+When present the metadata_crop OBU applies starting at the next frame in the sequence 
+with matching temporal_id and spatial_id, and shall apply to all matching frames until the next Key Frame
+or metadata_crop OBU. The output picture should be cropped to the region as specified in the 
+cropping metadata OBU. When applied, the crop shall be after all normal decode operations as a 
+post-processing step (after film grain synthesis in the output process).  This metadata has no 
+effect on the decoding process.
+
+**crop_x_offset** specifies the minimum pixel column containing picture data which should be rendered.
```

Comment:
Minor editorial detail, but maybe it would be better to specify them in the same order as in the syntax?

Reply:
That is a good point, yes I will re-order.